File: abc/abc307/a.py
# https://atcoder.jp/contests/abc307/submissions/42942091

n = int(input())
a = list(map(int, input().split()))

cnt = 0
sum = 0
for i in range(len(a)):
  cnt += 1
  sum += a[i]
  
  if cnt == 7:
    print(sum, end = ' ')
    sum = 0
    cnt = 0

File: abc/abc307/a.cs
// https://atcoder.jp/contests/abc307/submissions/46508692

using System;

public class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());
    string[] t = Console.ReadLine().Split(" ");

    int sum = 0;
    for(int i = 0; i < t.Length; i++)
    {
      sum += int.Parse(t[i]);
      if(i % 7 == 6)
      {
        Console.Write(sum + " ");
        sum = 0;
      }
    }
  }
}

